modules/slowzones/map/DirectionIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import type { SlowZoneDirection } from './segment';
+
+import styles from './DirectionIndicator.module.css';
+
+type Props = {
```

Comment:
This should be an interface, and be more specifically named
```suggestion
interface DirectionIndicatorProps {
```